docs/src/pages/styles/api/api.md

```diff
@@ -6,18 +6,18 @@
 
 A function which returns [a class name generator function](https://cssinjs.org/jss-api/#generate-your-class-names).
 
-#### Arguments
+### Arguments
```

Comment:
The motivation was to remove these values from the table of contents. What about?

```suggestion
**Arguments**
```

Reply:
Why do you want them removed from the TOC. There obviously a sub-item of the method.

Reply:
What do you think of the difference of experience between 

A.
<img width="179" alt="Capture d’écran 2020-04-02 à 18 56 31" src="https://user-images.githubusercontent.com/3165635/78276708-b6355b80-7513-11ea-9e9b-584217c9d44c.png">

B.
<img width="184" alt="Capture d’écran 2020-04-02 à 18 56 40" src="https://user-images.githubusercontent.com/3165635/78276715-b897b580-7513-11ea-9941-12f39149fbf3.png">

Are "Arguments", "Returns", "Example" important enough to be included in the TOCs? It comes with the assumption that the longest the TOCs is, the harder it's to scan and find interesting content.

Reply:
On a side note, the change goes against the other header structure in the documentation. So far The `#### Arguments` level was used exclusively. It would be awesome to uniformized it.

Reply:
I am not able to follow. You obviously want to group the content and describe it with a single word. That is a heading. Not sure what we're discussing here.

Reply:
> It would be awesome to uniformized it.

I was wrong, the header structure in the other pages that have `Arguments` as a header already have a continuous `<hx>` structure.